### PR TITLE
Add icons to "Add goal".

### PR DESCRIPTION
**Pull request by Zarabadoo**
- Preface the text with "+".
- Add a "x" to the right of the title bar when the card is expanded.

### CODE REVIEW

File: css/acquia_lift.admin.css [hunk 1]
```diff
@@ -300,6 +300,31 @@ tr.acquia-lift-report-control {
...
+  width: 1.4rem;
+  height: 1.4rem;
+  line-height: 1em;
+  vertical-align: text-bottom;
+}
+.acquia-lift-campaign-add .el-card .el-card__header .el-card__title[aria-controls]:before {
+  content: url('../images/actions__add.svg');
+}
+.acquia-lift-campaign-add .el-card .el-card__title:after {
+  content: url('../images/actions__close.svg');
```
Comment (eshtadc) [inline css/acquia_lift.admin.css:320]: I think any other spot that we've used svg graphics we've still kept a fallback to png - however looking at the support map I guess that isn't really necessary anymore :-)

**Comment by eshtadc** [comment]
Looks ready to merge into workflow.  Thanks!

**Comment by eshtadc** [comment]
Forgot - you can use https://www.drupal.org/node/2430417 in the commit message.  It's kind of a catch-all story.

File: css/acquia_lift.admin.css (hunk 1 above)
Comment (Zarabadoo) [inline css/acquia_lift.admin.css:320]: The admin interfaces should be fine with svg only. The audience will not see these images.

**Comment by eshtadc** [comment]
This has been merged into workflow along with a slew of other user-facing improvements.

**Pull request closed without merging**